bicycleparameters/main.py

```diff
@@ -532,14 +537,18 @@ def plot_bicycle_geometry(self, show=True, pendulum=True,
         except AttributeError:
             pendulum = False
 
-        fig, ax = plt.subplots()
+        fig1 = go.Figure()
+        # fig, ax = plt.subplots()
```

Comment:
I think making a new `plot_bicycle_geometry_plotly()` method or `plot_bicycle_geometry(use_plotly=True)` is better. We want to keep this existing method working because it is still useful for standard use cases.

Reply:
A new method is better, because these methods are way too long already. A good rule of thumb for function/method length is 30 lines or so.